src/main/kotlin/app/revanced/patches/youtube/layout/spoofappversion/bytecode/patch/SpoofAppVersionPatch.kt

```diff
@@ -54,7 +54,7 @@ class SpoofAppVersionPatch : BytecodePatch(
                     "revanced_spoof_app_version_target_entries",
                     listOf(
                         StringResource("revanced_spoof_app_version_target_entry_1", "17.30.35 - Restore old UI layout"),
-                        StringResource("revanced_spoof_app_version_target_entry_2", "17.01.35 - Enable sorting videos by oldest"),
+                        StringResource("revanced_spoof_app_version_target_entry_2", "17.01.35"),
```

Comment:
This version has been kept as "Some differences" are present. This may be mentioned here. Otherwise, the user would wonder why this version exists or what it does.

Reply:
I'd say including at least the information on the slight difference for watch history is better than nothing. Someone coming across this list will wonder why this version is present otherwise. Reasoning its existence with at least something is a good idea.